docs/proposals/1199-multitenant-api-proposal/README.md

```diff
@@ -0,0 +1,250 @@
+	Group Group `json:"group,omitempty"`
+
+	// Kind is kind of the referent. For example "InferencePool".
+	//
+	// +optional
+	// +kubebuilder:default="InferencePool"
+	Kind Kind `json:"kind,omitempty"`
+
+	// Name is the name of the referent.
+	//
+	// +kubebuilder:validation:Required
+	Name ObjectName `json:"name"`
+}
+
+
+type PolicyProfile struct {
+	metav1.TypeMeta
+    metav1.ObjectMeta
+
+    Spec PolicyProfileSpec
+}
+
+type PolicyProfileSpec struct {
+	// this is a departure from InferenceModel that used string for criticality.
+	// We got quite a bit of feedback around allowing for custom criticality bands, so an int/enum is more flexible & carries inherent stack rank value.
+	Criticality *int
+	FairnessPolicy NotYetDefinedFairnessPolicy
+	SLOPolicy InferenceSLOPolicy
+}
+
+type InferenceCriticalityPolicy struct {
+    Criticality int32
+}
+```
+
+### Intent
+
+The purpose(s) of the `InferenceObjectives` is:
+- Create a strong concept of a tenant within the inference pool, used to associate groups of requests together for the purpose of Flow Clontrol - which can enforce:
+  - Fair resource sharing
+  - Inter-tenant prioritization
+  - SLO attainment
+- Create a handle with which to attach scheduling policies, allowing for heterogenous scheduling behavior across tenants 
+  - Such as the [InferenceSLOPolicy](https://docs.google.com/document/d/1j2KRAT68_FYxq1iVzG0xVL-DHQhGVUZBqiM22Hd_0hc/edit?resourcekey=0-5cSovS8QcRQNYXj0_kRMiw&tab=t.0#heading=h.emkaixupvf39).
+- Detach identification from the modelName field
+
+## Usage
+
+The InferenceObjectives API surface has 3 general scopes:
+- Identification
+- Pool Binding
+- Policy Handle
+
+### Pool Binding
+
+Tackling the simplest first; the single Pool Binding field `PoolRef` is simply to associate a given InferenceObjectives object with a pool. Meaning that InferenceObjectives with duplicate ID's across different pools are considered valid.
+
+### Identification
+**Note**: This ID field is proposed for Phase 2, & Phase 1 will use the kube name as the identifier in the short term. To make a smooth transition, the ID field would default to the kube name.
+
+The only field associated with identification is the `ID` field. A unique ID field was chosen (rather than using the metadata name), because:
+- We do not want to put the same restrictions on the string that is enfored on a kube resource name
+- The ID name may be duplicated across different pools
+- Use of a kube-generated name would force an upstream Auth mechanism to be aware of the `InferenceObjectives` API
+
+This ID can be any string; in the case of a MaaS platform, it may identify a customer via a one-way hashed JWT. Or for sharing an InferencePool between teams, it may be a team-id. 
+
+***Important***: In order to support a high volume of tenants, by default IGW will _allow_ unique IDs that do not have an explicit InferenceObjectives object defined. Instead the default for all policies will be used.
```

Comment:
Unclear on first read. What is the "default for all policies" that will be in use?
For requests having an ID which is not matched, where/how is the default policy defined?

Also, do we differentiate between no identity (e.g., missing header, perhaps implying "anonymous"?) and a request with identity that does not have a corresponding matching ID in any policy (e.g., apply default pool policies to it)?